object/src/file.rs

```diff
@@ -103,7 +103,7 @@ impl<D, T> OpenFileOptions<D, T> {
     }
 
     /// Set the transfer syntax index to use when reading the file.
-    pub fn tranfer_syntax_index<Tr>(self, ts_index: Tr) -> OpenFileOptions<D, Tr>
+    pub fn transfer_syntax_index<Tr>(self, ts_index: Tr) -> OpenFileOptions<D, Tr>
```

Comment:
Changing the name of this public function would be a breaking change now. We can 1) postpone this change until the time we aim for 0.9, or 2) re-add `tranfer_syntax_index` as a deprecated function which calls `transfer_syntax_index` underneath.

Reply:
Will re-add as deprecated.  I should have thought of that!